ansible/kubernetes/roles/common/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: download k3s installation script locally.
+  delegate_to: localhost
+  become: false
+  ansible.builtin.get_url:
+    url: https://get.k3s.io
+    dest: "./k3s_install.sh"
+    mode: "a+x"
```

Comment:
Since this is happening on the local host and it's a dependency of multiple roles, would it make sense to add `run_once: true` to this?